hw_diag/diagnostics/gateway_diagnostics.py

```diff
@@ -0,0 +1,26 @@
+from hm_pyhelper.diagnostics import DiagnosticsReport
+from hm_pyhelper.diagnostics.diagnostic import Diagnostic
+from hw_diag.utilities.miner import fetch_miner_data
+from hm_pyhelper.miner_param import LOGGER
+import grpc
+
+
+class GatewayDiagnostic(Diagnostic):
+    KEY = 'GATEWAY'
```

Comment:
There is no need to make KEY and FRIENDLY_NAME different. The reason we have two values is to be backwards compatible with logic that relies on old two letter keys like `E0` being present. For net-new entries that we add to the diagnostics report, we should make KEY and FRIENDLY_NAME the same.

Reply:
fixed.